src/Lesson11/Task5.java

```diff
@@ -0,0 +1,30 @@
+package Lesson11;
+
+public class Task5 {
+    String text;
```

Comment:
Why do u declare this variable on the class layer if, u use it only in the method?